tests/thingpedia-integration/k8s/manager-config/developer-deployment.json

```diff
@@ -0,0 +1,69 @@
+{
```

Comment:
It would be nicer if this was YAML instead of JSON, given it's a k8s resource. Is there any way to do that?